sklearn/utils/_array_api.py

```diff
@@ -34,8 +34,8 @@ def yield_namespace_device_dtype_combinations():
         # tests are regular numpy arrays without any "device" attribute.
         "numpy",
         # Stricter NumPy-based Array API implementation. The
-        # numpy.array_api.Array instances always a dummy "device" attribute.
-        "numpy.array_api",
+        # array_api_strict.Array instances always a dummy "device" attribute.
```

Comment:
```suggestion
        # array_api_strict.Array instances always have a dummy "device" attribute.
```
That word seems to have been missing since a while